go/app/infra.go

```diff
@@ -2,21 +2,28 @@ package app
 
 import (
 	"context"
-	"encoding/json"
+	"database/sql"
 	"errors"
 	"fmt"
 	"os"
-	// STEP 5-1: uncomment this line
-	// _ "github.com/mattn/go-sqlite3"
+
+	_ "github.com/mattn/go-sqlite3"
 )
 
 var errImageNotFound = errors.New("image not found")
 
 type Item struct {
-	ID       int    `db:"id" json:"-"`
-	Name     string `db:"name" json:"name"`
-	Category string `db:"category" json:"category"`
-	Image    string `db:"image" json:"image"`
+	ID           int    `db:"id" json:"-"`
+	Name         string `db:"name" json:"name"`
+	CategoryID   int    `db:"category_id" json:"category_id"`
+	CategoryName string `db:"category" json:"category"`
```

Comment:
```suggestion
	CategoryName string `db:"category" json:"category_name"`
```

For best practice, we have CategoryName -> category_name similar to categoryID